client/my-sites/upgrades/checkout/supporting-text.jsx

```diff
@@ -6,6 +6,7 @@ var React = require( 'react' );
  * Internal dependencies
  */
 var cartValues = require( 'lib/cart-values' ),
+	support = require( 'lib/support' ),
```

Comment:
I like `client/lib/external-links/support` :+1: 
My issue was with the name of the import too (`support`). So for this new location, something less generic than `support` should be used - like `supportLinks` or `externalSupportLinks` if we want to be extra verbose.